apps/web/public/static/locales/en/common.json

```diff
@@ -836,7 +836,7 @@
   "connect_metamask": "Connect Metamask",
   "create_events_on": "Create events on",
   "enterprise_license": "This is an enterprise feature",
-  "enterprise_license_description": "To enable this feature, get a deployment key at {{consoleUrl}} console and add it to your .env as CALCOM_LICENSE_KEY. If your team already has a license, please contact {{supportMail}} for help.",
+  "enterprise_license_description": "To enable this feature, have an administrator go to {{setupUrl}} to enter a license key. If a license key is already in place, please contact {{supportMail}} for help.",
```

Comment:
Changed the wording given the fact we moved out from environment variable to the new db model through the admin wizard.

Reply:
Not restoring this so we can rely on the admin wizard steps for people to have a better experience regarding license key

Reply:
this was a mistake. we need to keep `get a deployment key at {{consoleUrl}} console and`